pkgs/top-level/all-packages.nix

```diff
@@ -12573,6 +12575,10 @@ in
   libappindicator-gtk3 = libappindicator.override { gtkVersion = "3"; };
   libappindicator = callPackage ../development/libraries/libappindicator { };
 
+  libayatana-appindicator-gtk2 = libayatana-appindicator.override { gtkVersion = "2"; };
+  libayatana-appindicator-gtk3 = libayatana-appindicator.override { gtkVersion = "3"; };
+  libayatana-appindicator = callPackage ../development/libraries/libayatana-appindicator { };
```

Comment:
Nit: I think it would make more sense to move the "base" package above the others (despite `libappindicator` not doing that just a few lines above).